assets/js/atomic/utils/index.js

```diff
@@ -130,14 +104,9 @@ export const getProductLayoutConfig = ( innerBlocks ) => {
 		return DEFAULT_PRODUCT_LIST_LAYOUT;
 	}
 
-	const componentLookup = BLOCK_MAP.reduce(
-		( acc, it ) => ( ( acc[ it.blockName ] = it ), acc ),
-		{}
-	);
-
 	return innerBlocks.map( ( block ) => {
 		return {
-			component: componentLookup[ block.name ].component,
+			component: BLOCK_MAP[ block.name ].name,
```

Comment:
Do we need error handling here to ensure the key exists? Isset? Try/catch?

Reply:
Added a check in 0a43aea6dafaf0d44ee55bec108b93fafdb631ae.